src/main/scala/pipelined/cpu-bp.scala

```diff
@@ -387,6 +391,9 @@ class PipelinedCPUBP(implicit val conf: CPUConfig) extends Module {
   mem_wb.readdata   := io.dmem.readdata
   mem_wb.wbcontrol  := ex_mem.wbcontrol
 
+  // Stall pipeline if neither instruction nor data memory are ready
+  val memStall = ~(io.imem.good && io.dmem.good)
```

Comment:
Required for this design?

Reply:
You're correct, I'll rebase this line out.